wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -205,11 +205,15 @@ instance ToDocs Account where
   readOnlyFields   = readOnlyFieldsFromJSON
   descriptionFor _ = "An Account."
 
+instance ToDocs WalletAddress where
+  readOnlyFields   = readOnlyFieldsFromJSON
+  descriptionFor _ = "An Address."
```

Comment:
Maybe worth expanding saying it's "An address plus some useful info related to it?"